lib/brewdler/brew_dumper.rb

```diff
@@ -0,0 +1,42 @@
+require "json"
+require "version"
```

Comment:
How about we do the version sort in the homebrew, so we only need use the last one from the JSON output?